python/tvm/relay/__init__.py

```diff
@@ -10,6 +10,7 @@
 from .op import Op
 from .op.tensor import *
 from .op import nn
+from .op import transform
```

Comment:
We only need namespace nn, vision. Normal numpy API should be in the root namespace